### PR TITLE
Forwarded request's actor fix

### PR DESCRIPTION
**Pull request by MahatiC**

### CODE REVIEW

File: tests/infra/logging_app.py [hunk 1]
```diff
@@ -104,6 +104,7 @@ def issue(
         msg=None,
         user=None,
         url_suffix=None,
+        private_url=None,
```
Comment (jumaffre) [inline tests/infra/logging_app.py:107]: I think this should be called `url`, or in fact `uri` in this case as this doesn't include the full address?

File: tests/infra/logging_app.py [hunk 2]
```diff
@@ -138,6 +139,7 @@ def issue(
                     if self.scope is not None:
                         args["scope"] = self.scope
                     url = "/app/log/private"
+                    url = private_url if private_url else url
```
Comment (jumaffre) [inline tests/infra/logging_app.py:142]:
```suggestion
                    url = url or "/app/log/private"
```

**Review by jumaffre — APPROVED** [review]

File: tests/infra/logging_app.py (hunk 1 above)
Comment (MahatiC) [reply tests/infra/logging_app.py:107]: Just went with the existing convention in the current test i.e. `url`

File: tests/infra/logging_app.py (hunk 2 above)
Comment (MahatiC) [reply tests/infra/logging_app.py:142]: I believe this suggestion is applicable only if I change the param name to `url` from `private_url`? (Other than replacing the if else semantic)

Comment (jumaffre) [reply tests/infra/logging_app.py:142]: Yes, this is just a "Python can do it in one line" comment :)

Comment (MahatiC) [reply tests/infra/logging_app.py:142]: Gotcha, will remember this for next time :)

**Review by DomAyre — APPROVED** [review]

**Review by eddyashton — APPROVED** [review]

**Pull request merged**